apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -822,7 +822,15 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
         let response: ApplicationListInterface = null;
 
         try {
-            response = await getApplicationList(null, null, "name eq " + value.toString());
+            let filter: string = "name eq ";
+
+            if (value.toString().includes(" ")) {
+                filter += "\"" + value.toString() + "\"";
```

Comment:
Shall we use template literals. Check `getFilterValue` func as well. :)

```js
`\${value.toString() }\`;
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5246/commits/30a1c1593cfaf8a3ecc56558d84af1b37010c26d